src/test/java/jenkins/branch/CustomOrganizationFolderDescriptorTest.java

```diff
@@ -116,17 +116,21 @@ public void insideFolder() throws Exception {
         public void visitSources(SCMSourceObserver observer) throws IOException, InterruptedException {}
     }
     private static abstract class MockNavigatorDescriptor extends SCMNavigatorDescriptor {
+
```

Comment:
I guess you did not know to

    git checkout master -- src/test/java/jenkins/branch/CustomOrganizationFolderDescriptorTest.java